edgelet/contrib/enterprise-linux/aziot-edge.spec

```
@@ -22,7 +22,8 @@ URL:            https://github.com/azure/iotedge
 %{?systemd_requires}
 BuildRequires:  systemd
 Requires(pre):  shadow-utils
-Requires:       aziot-identity-service = 1.4.4-1%{?dist}
+Requires:       (moby-engine or docker-ce)
+Requires:       aziot-identity-service = 1.4.0~dev-1%{?dist}
```

We don't want the 1.4 release of aziot-edge to depend on the dev build of aziot-identity-service, do we?

```suggestion
Requires:       aziot-identity-service = 1.4.4-1%{?dist}
```